litmus/AArch64Compile_litmus.ml

```diff
@@ -431,7 +441,32 @@ module Make(V:Constant.S)(C:Config) =
           memo=sprintf "%s %s,%s" memo f1 f2;
           inputs = r2; outputs=r1; reg_env=add_q (r1@r2);}
 
+    let do_mov memo v rd k os = match v, k, os with
```

Comment:
Nipicking:  "do_movz" would be a better name.

Reply:
will fix